usercp.php

```diff
@@ -243,13 +243,21 @@
 		}
 
 		if($cfield == 'icq')
-		{
-			$user[$cfield] = $mybb->get_input($cfield, 1);
-		}
-		else
-		{
-			$user[$cfield] = $mybb->get_input($cfield);
-		}
+        {
+            $user[$cfield] = $mybb->get_input($cfield, 1);
+            if(my_strlen($user[$cfield]) > 10)
+            {
+                error($lang->contact_field_icqerror);
+            }
+        }
+        else
+        {
+            $user[$cfield] = $mybb->get_input($cfield);
+            if(my_strlen($user[$cfield]) > 75)
+            {
+                error($lang->contact_field_error);
+            }
+        }
```

Comment:
Indentation should be tabs:

```suggestion
		{
			$user[$cfield] = $mybb->get_input($cfield, 1);

			if(my_strlen($user[$cfield]) > 10)
			{
				error($lang->contact_field_icqerror);
			}
		}
		else
		{
			$user[$cfield] = $mybb->get_input($cfield);

			if(my_strlen($user[$cfield]) > 75)
			{
				error($lang->contact_field_error);
			}
		}
```